sat/cli/bootsys/state_recorder.py

```diff
@@ -205,10 +204,7 @@ def dump_state(self):
             raise StateError(f'Failed to write state to file {local_new_file_name}: {err}') from err
         try:
             LOGGER.debug('Uploading %s to S3', new_file_name)
-            # TODO(SAT-926): Start verifying HTTPS requests
-            with warnings.catch_warnings():
-                warnings.filterwarnings('ignore', category=InsecureRequestWarning)
```

Comment:
There are some similar `filterwarnings` calls in `sat.apiclient.ims.IMSClient` in the methods for renaming images. I imagine we could similarly get rid of the filtering there too now that we only print one warning for this. I might make that change and test it out with `sat bootprep`.

Reply:
Here is the gist with the test results from running `sat bootprep` against an input file that creates an image by configuring an existing image. This exercises the image rename code that uses the S3 API directly. As expected, there is just one warning despite multiple calls to S3 APIs. 

https://gist.github.com/haasken-hpe/5f7b003f1866b75438d3fd60974d43d3